website/_enemies/forest/totem-large.md

```diff
@@ -11,3 +11,7 @@ massive: true
 hp: 12
 game_name: enemy_stoneboss
 ---
+
+Attacks by falling on the enemy. Has an active hitbox as long as it is falling during its attack, which allows the player to farm super quickly as demonstrated in top [Overgrown Temple](#maps-level-overgrown-temple) Any% replays.
+
+Can be lured outside of its path.
```

Comment:
Being able to be lured outside of its path needs to be a dedicated stat. See [CONTIBUTING.md#enemies](https://github.com/gonX/dustforce-wiki/blob/master/CONTRIBUTING.md#enemies)

Reply:
Solving personally